setup.py

```diff
@@ -54,6 +54,7 @@
     ],
     extras_require={
         'dev': [
+            'coverage',
```

Comment:
Please make separate extras_require definition for the ci. For example, the ci does not need libsass or sphinx, and the developer does not need coverage, etc.